ci/courtesy-push.yml

```diff
@@ -0,0 +1,55 @@
+steps:
+- task: NuGetToolInstaller@0
+  inputs:
+    versionSpec: '5.4.0'
+  displayName: 'Install nuget'
+
+- task: DownloadBuildArtifacts@0
+  inputs:
+    artifactName: IndividualNugetPackages
+    downloadPath: 'IndividualNugetPackagesDownloaded'
+  displayName: 'Download Artifact'
+
+- script: dir
+
+- script: node azure-pipelines-tasks\ci\courtesy-push\courtesy-push.js AzureDevOps\.nuget\externals\UnifiedDependencies.xml IndividualNugetPackagesDownloaded\IndividualNugetPackages\unified_deps.xml
+  displayName: 'Update unified deps'
+
+- task: NuGetAuthenticate@0
+  inputs:
+    nuGetServiceConnections: 'Codex-Deps'
+  displayName: 'Authenticate with nuget'
```

Comment:
Is the safety block here having queue ability on the pipeline? We need to be very careful with what the gate is and who can use it.

Reply:
Yes, but I also don't know that we need to be _that_ careful. We won't run it as CI or anything, beyond that it doesn't give away any permissions any MSFT contributor doesn't already have. Skyrise automatically gives you all of this